libs/input-mapping/src/Helper/RealDevStorageTableRewriteHelper.php

```diff
@@ -45,27 +46,49 @@ public function rewriteTableStatesDestinations(
     }
 
     private function rewriteSourceBranchId(
-        string $source,
+        InputTableOptions $tableOptions,
         ClientWrapper $clientWrapper,
         LoggerInterface $logger,
     ): array {
+        $source = $tableOptions->getSource();
+        $defaultBranchId = $clientWrapper->getDefaultBranch()->id;
+
+        if ($tableOptions->getSourceBranchId() !== null) {
+            // use source branch from mapping definition
+            $branchId = (string) $tableOptions->getSourceBranchId();
+            $logger->info(sprintf(
+                'Using input "%s" from %s branch "%s".',
+                $source,
+                $branchId === $defaultBranchId ? 'main' : 'dev',
```

Comment:
tak většinou tomu říkáme default

Reply:
ne vsude https://github.com/keboola/platform-libraries/pull/158/files#diff-f4ecfae2b9920a10bb469c21b0e57e72efafdffbdcf5a69fc18e01c470b120e2L54

Reply:
tak ja to prejmenuju

Reply:
njn, je v tom chaos, v UI se to pro jistotu jmenuje "Production" takže babo raď

Reply:
sjednotil jsem ten naming na default